### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token 'export'

### DISCUSSION THREAD

**Discussion by fyrak1s** [plausible/analytics#1703] [Self-Hosted Support] [answered]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Describe the bug

When I go to the page "Waiting for first pageview" and go into the developer console, I am getting the following error in dev console:
```
Uncaught SyntaxError: Unexpected token 'export'
```

In `fetch.min.js` at:
```js
export function Headers(e) {
    this.map = {},
    e instanceof Headers ? e.forEach(function(e, t) {
        this.append(t, e)
    }, this) : Array.isArray(e) ? e.forEach(function(e) {
        this.append(e[0], e[1])
    }, this) : e && Object.getOwnPropertyNames(e).forEach(function(t) {
        this.append(t, e[t])
    }, this)
}
```

Happens on my self-hosted instance.

### Expected behavior

Should not have any error message.

### Environment

```markdown
- Browser: Brave
- Browser Version: 1.35.103
```

**Comment by fyrak1s** [comment]
Hey, this was actually meant as a bug report.

**Comment by fyrak1s** [comment]
Anything?

**Comment by ukutaht** [comment]
I don't see any `fetch.min.js` file anywhere? How did you find it?

**Reply by fyrak1s** [reply]
It is in the developer console in the browser.

**Comment by Vigasaurus** [comment, marked as answer]
I just checked on mine too (as I remembered seeing the same error a while back) - the error does occur, but the originating script is a Cloudflare script, not anything from Plausible. I'm think it's caused by some interaction between Phoenix and CF's rocket-loader, but it's a hyper-specific Cloudflare issue. 

This is the file it's referencing: https://cdnjs.cloudflare.com/ajax/libs/fetch/3.0.0/fetch.min.js

**Reply by fyrak1s** [reply]
I do not use Cloudflare though 🤔

Is plausible working as expected for you?

**Reply by Vigasaurus** [reply]
Yep, works perfectly fine. I assume the fetch script isn't necessarily super CF specific (it seems to just be a somewhat generic fetch implementation served up by a CF CDN). Either way, the error has been inconsequential to my usage of Plausible (it's been showing up for at least the past 8 months that I know of, and I assume it'd be just as inconsequential to your usage of it too.
